### PR TITLE
Fix typo in FAQ

### PR DESCRIPTION
**Pull request by jirisvd**
Adds missing word in FAQ answer (`once you see how powerful it is and [how] rapid your development becomes`)

### CODE REVIEW

File: classes/class_compositoreffect.rst
Comment (AThousandShips) [inline classes/class_compositoreffect.rst:1]: These changes needs to be done over [here](https://github.com/godotengine/godot), changing the generated files is not allowed, see [here](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html)

**Review by AThousandShips — CHANGES_REQUESTED** [review]

File: classes/class_compositoreffect.rst
Comment (jirisvd) [reply classes/class_compositoreffect.rst:1]: My bad, I force pushed a new commit so that only the FAQ fix remains, should be good now

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Merged. Thanks and congrats on your first merged contribution!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.3 in https://github.com/godotengine/godot-docs/pull/10038.

**Pull request merged**